tests/Unit/PaymentTest.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace Tests\Unit;
+
+use Tests\TestCase;
+use App\Helpers\Payment\EventPayment;
+use App\Event;
+use App\Participant;
+
+class PaymentTest extends TestCase
+{
+
+    private $payment;
+    private $event;
+    private $participant;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        $this->event = Event::findOrFail(1);
+        $this->participant = Participant::findOrFail(1);
+
+        $this->payment = (new EventPayment())
+            ->event($this->event)
+            ->participant($this->participant)
+            ->existing(true);
+    }
+
+    /**
+     * A basic unit test example.
```

Comment:
Deze phpdoc updaten of verwijderen :)

Reply:
Oef, slordig, will do. Ik zal alle php docs ff nalopen anders